docs/contributing/Pull_Requests.mdx

```diff
@@ -84,6 +103,14 @@ Once you've addressed all our feedback by making code changes and/or started a f
 
 Once the feedback is addressed, and the PR is approved, we'll ensure the branch is up to date with `main`, and merge it for you.
 
+#### Updating Over Time
+
+You generally don't need to keep merging commits from `main` into your PR branch.
```

Comment:
You might mention if there's a preference for "merge" vs "rebase" if merging from main is done. 

I come from an environment where we'd always rebase against main and force push for this purpose, but it seems like merging from main is preferred in this codebase.

Reply:
This is already mentioned at the top of this page:

> Please don't:
>
> - Force push after opening a PR
>   - Reasoning: GitHub is not able to track changes across force pushes, which makes it take longer for us to perform incremental reviews

Reply:
Perhaps this is just a weird silo I'm in, but I work in an environment where

1. Thou shalt not create a merge commit
2. Though shalt not force push
3. 1. Takes precedence over 2. if there is a merge conflict. 

So, having read the contributing guide, I still, intentionally, force pushed the first time I had to resolve a merge conflict with main for a PR to typescript-eslint, thinking that was the right thing to do. 🤷‍♂️

Reply:
Hmmm, that does sound like a weird catch 22. I'm personally in the "always force push" camp. One sentence saying "merge not rebase" sounds good